client/app/dashboard/dashboard.controller.js

```diff
@@ -1,4 +1,8 @@
 angular.module('OpenEXP')
-    .controller('DashboardCtrl', ['$scope', ($scope) => {
+    .controller('DashboardCtrl', ['$scope', 'boardFactory', 'simulatorFactory', ($scope, boardFactory, simulatorFactory) => {
```

Comment:
Believe we have ng-annotate loading in (I'll have to test it to make sure it is working) but the whole point of putting these definitions in an array is for when it is minified it won't lose it's reference to $scope, board factory, etc. i.e it won't minify those keywords that we need to stay the same. So what you have here is fine, just wanted to say that I'll probably be modifying it so we don't have to do this anymore as it's a pain in the ass.